### ISSUE TITLE
Android app fails to download library tree

### ISSUE THREAD

**Issue by tweedbeats** [clementine-player/Clementine#4792]
I built Clementine from source on my Pi B+, running the latest Raspbian, and latest firmware. 
This is the guide that I followed:
https://github.com/clementine-player/Clementine/wiki/Clementine-on-Raspberry-Pi

What I find is that if I add a low number of audio tracks in the library - say 2000, the program works and the Android app downloads the library tree properly. Everything works ok. However, upon adding one of my "other" folders containing around 9000 tracks into the library, the Android app now fails to sync/download/show the library tree.

I receive an error on the app and it says "Could not download library. Download canceled".
I have tried to delete the app data and restart the process to fail again.

Here's the error from the Raspbian terminal/console.
04:46:26.463 DEBUG unknown                          Downloader true  
04:46:26.464 INFO  OutgoingDataCreator:280          No current item found! 
04:46:31.579 DEBUG unknown                          Client disconnected

**Comment by amuttsch** [comment]
Fixed in Clementine Remote.

**Comment by tweedbeats** [comment]
Great work, thanks! Another thing I noticed is actually something that is written in the guide I provided.

These commands here:
git clone https://github.com/clementine-player/Clementine.git && cd Clementine
git checkout release-1.2
cd bin

They don't really give clues about where Clementine should be placed (/usr?), and also, there's no "bin" directory under the "Clementine" directory initially. I created it manually to continue the process, so I'm not sure if this is an unintended omission in the guide, or a different bug altogether.
